src/fusor/fusor.py

```diff
@@ -266,55 +264,53 @@ async def transcript_segment_element(
                 )
                 _logger.warning(msg)
                 return None, [msg]
-            residue_mode = kwargs.get("residue_mode")
-            # TODO: Remove once fixed in cool_seq_tool - need to verify that this code isn't still needed
-            if residue_mode != ResidueMode.INTER_RESIDUE:
-                start = kwargs.get("start")
-                kwargs["start"] = start - 1 if start is not None else None
-                kwargs["residue_mode"] = "inter-residue"
             chromosome = kwargs.get("chromosome")
-            # if chromosome is a string, assume it's an accession, fix it for the kwargs since CST expects this as alt_ac
+            # if chromosome is a string, assume it's an accession, fix it for the kwargs since CST expects this as genomic_ac
```

Comment:
Shouldn't chromosome always be a string?

Reply:
I think there was possibly a time where we used to accept something like`12` for chromosome 12 instead. From the fusion-curation interface, we only ever send a prepopulated accession though. This isn't something the user can even change, so it may make sense to simplify this